.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,56 @@
+name: Zulip Desktop CI
+
+on: [push]
+
+jobs:
+  test:
+    name: Test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: 
+        - ubuntu-latest
+        - windows-latest
+#         - macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setting up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '*'
```

Comment:
Here, let's do `12.x` here. The use of `*` sounds like it will use beta version of node which we should avoid.

Reply:
Yes, 12.x will use the recommended version.